### ISSUE TITLE
Rajouter bouton copie de la dénomination de voie depuis Pifometre

### ISSUE THREAD

**Issue by gendy54** [osm-fr/osm-vs-fantoir#451]
Hello,

Dans le même genre que #431 et qui aide bien, si on pouvait rajouter un bouton de copie du nom de la voie depuis Pifomètre, ce serait super.

Je le vois juste à droite de la colonne Libellé BAN ou cadastre

**Comment by Khomille** [comment]
Ouais, y a moyen. Mais sans le message de confirmation "copié" alors, ça prend trop de place. Juste le picto. 
Faut faire des tests et voir si ça rentre, et si c'est pas trop chargé.

**Comment by gendy54** [comment]
Oui, juste en coloriant en vert le logo de copie, histoire qu'on voit quand même que la manip s'est bien faite. Ca suffira.

**Comment by Khomille** [comment]
C'est fait !

**Comment by gendy54** [comment]
Oh quel bonheur cette utilisation. Il va falloir maintenant que je désapprenne à couloir faire copier/coller mais surtout copier.
Fini les espaces avant et après le copiage !

**Comment by gendy54** [comment]
Vœu exaucé, ticket clôturé :)
